wpeverest/everest-forms
# Fix - Payment checkbox smart tag value is not showing email.

### All Submissions:


### Changes proposed in this Pull Request:
Previously, the Payment checkbox smart tag value is not showing the email.


### How to test the changes in this Pull Request:

1. Create a form.
2. Drag and drop the Payment checkbox field.
3. Add the Payment checkbox smart tag in email content and verify whether the value is showing or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Payment checkbox smart tag value is not showing email.
